hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestReadRetries.java

```diff
@@ -95,19 +88,6 @@
       storageContainerLocationClient;
 
   private static final String SCM_ID = UUID.randomUUID().toString();
-  private String bucketLayout;
-
-  public TestReadRetries(String bucketLayout) {
-    this.bucketLayout = bucketLayout;
-  }
-
-  @Parameterized.Parameters
-  public static Collection<Object[]> data() {
-    return Arrays.asList(
-        new Object[]{OMConfigKeys.OZONE_DEFAULT_BUCKET_LAYOUT_DEFAULT},
-        new Object[]{OMConfigKeys.
-              OZONE_BUCKET_LAYOUT_FILE_SYSTEM_OPTIMIZED});
-  }
```

Comment:
This is a file system related test, which is no longer valid for bucket layouts other than FSO since we are in process of removing the `OZONE_OM_ENABLE_FILESYSTEM_PATHS` configuration key.